### ISSUE TITLE
Wrong error message on `graphcool init -n`

### ISSUE THREAD

**Issue by RomanGotsiy** [graphcool/graphcool-cli#130]
**Steps to reproduce:**
```
$ graphcool init -n "Name With Space"
✖  Error: 'Name With Space' is not a valid project name. It must begin with an uppercase letter.
```

**Expected behavior:**
either error message saying project name can't contain spaces or no error at all

**Comment by marktani** [comment]
The expected behaviour is that there is no error message and the project is created with the name "Name With Space".

**Comment by timsuchanek** [comment]
Fixed in 1.3.13 beta
